vcpkg.json

```diff
@@ -4,14 +4,34 @@
     "builtin-baseline": "f14984af3738e69f197bf0e647a8dca12de92996",
     "dependencies": [
         "bzip2",
+        "catch2",
```

Comment:
Should this be added in a `tests` feature?

Reply:
Done.